services/ui-src/src/components/accordions/TemplateCardAccordion.tsx

```diff
@@ -68,8 +79,8 @@ const sx = {
     borderStyle: "none",
   },
   accordionButton: {
-    bg: "palette.gray_lightest",
     height: "3.5rem",
+    bg: "palette.gray_lightest",
```

Comment:
In some JSS blocks, I changed up the orders of things to match usual practices -- it's not a big thing, but it's a good thing to know because some shops are particular about it.